knowledge-portal/source/.gitlab-ci.yml

```diff
@@ -0,0 +1,51 @@
+copy-files:
+  stage: copy_to_central_repo
+  image: bitnami/git
+  script:
+    - ls -al .
+    # split repo path after DetectTechnologies
+    - arrIN=(${CI_REPOSITORY_URL//"DetectTechnologies/"/ })
```

Comment:
Can we generalize this to find the right split point even if the top-level namespace changes?